pkg/frontend/session.go

```diff
@@ -254,6 +255,9 @@ type BackgroundSession struct {
 func NewBackgroundSession(ctx context.Context, mp *mpool.MPool, PU *config.ParameterUnit, gSysVars *GlobalSystemVariables) *BackgroundSession {
 	ses := NewSession(&FakeProtocol{}, mp, PU, gSysVars)
 	ses.SetOutputCallback(fakeDataSetFetcher)
+	if stmt := trace.StatementFromContext(ctx); stmt != nil {
+		ses.uuid = stmt.SessionID
```

Comment:
re-use client connection's session_id as `BackgroundSession` uuid.  pls confirm that ok for `BackgroundSession` @daviszhen